### PR TITLE
Integration tests qa 222

### PR DESCRIPTION
**Pull request by LukasReschke**
From https://github.com/owncloud/core/pull/26219

### CODE REVIEW

File: build/integration/features/webdav-related.feature [hunk 1]
```diff
@@ -77,6 +77,28 @@ Feature: webdav-related
...
 
+	Scenario: Retrieving folder quota when quota is set and a file was uploaded
+		Given using dav path "remote.php/webdav"
+		And As an "admin"
+		And user "user0" exists
+		And user "user0" has a quota of "1 KB"
+		And user "user0" adds a file of 93 bytes to "/prueba.txt"
+		When as "user0" gets properties of folder "/" with
+		  |{DAV:}quota-available-bytes|
+		Then the single response should contain a property "{DAV:}quota-available-bytes" with value "600"
```
Comment (MorrisJobke) [inline build/integration/features/webdav-related.feature]: `Property "{DAV:}quota-available-bytes" found with value "592", expected "600" (Exception)`

File: build/integration/features/webdav-related.feature [hunk 2]
```diff
@@ -77,6 +77,28 @@ Feature: webdav-related
...
+		Given using dav path "remote.php/webdav"
+		And As an "admin"
+		And user "user0" exists
+		And user "user1" exists
+		And user "user1" has a quota of "1 KB"
+		And user "user0" adds a file of 93 bytes to "/user0.txt"
+		And file "user0.txt" of user "user0" is shared with user "user1"
+		When as "user1" gets properties of folder "/" with
+		  |{DAV:}quota-available-bytes|
+		Then the single response should contain a property "{DAV:}quota-available-bytes" with value "693"
```
Comment (MorrisJobke) [inline build/integration/features/webdav-related.feature]: `Property "{DAV:}quota-available-bytes" found with value "685", expected "693" (Exception)`

**Review by MorrisJobke — CHANGES_REQUESTED** [review]
Tests are failing

File: build/integration/features/webdav-related.feature (hunk 1 above)
Comment (nickvergessen) [inline build/integration/features/webdav-related.feature]: I guess it's because of welcome.txt which has a different size on our end

Comment (nickvergessen) [inline build/integration/features/webdav-related.feature]: https://help.nextcloud.com/t/which-tls-config-is-supported-by-the-clients/3708

Comment (rullzer) [inline build/integration/features/webdav-related.feature]: Yep it is because Nextcloud has a letter more than ownCloud

**Review by MorrisJobke — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
signed off check failed ;)

**Pull request merged**